### ISSUE TITLE
Allow fleet orders from MapDetailPanel with touchscreens

### ISSUE THREAD

**Issue by thewierdnut** [thewierdnut/endless-mobile#125]
Currently fleet orders are only possible using a right click, which is unavailable on touchscreen-only devices. 

Possibly a long press should do the trick.

**Comment by Arteneko** [comment]
By fleet orders, do you mean things like "Attack my target", "Harvest flotsam", and such?

**Comment by thewierdnut** [comment]
No, it means giving the fleet-jump commands from the map window. This is normally done on the desktop version with a right click.

**Comment by Arteneko** [comment]
oh right, then it’s available while in flight space through the fleet actions but not through the map indeed, gotcha
/ Artémis (it/its)
